plugin.video.tyt/main.py

```diff
@@ -19,247 +18,127 @@
-tytlive_url       = "/live/"
-tytlive_thumb     = "tytlive.jpg"
-members_live_url  = "/memberslive/"
-members_live_thumb = "tytlive.jpg"
-special_events_url = "/category/membership/specialevents/"
-special_events_thumb = "tytlive.jpg"
-allstar_url       = "/category/membership/all-star-tuesdays-membership/"
-allstar_thumb     = "allstar.jpg"
-members_cat = {"Hour 1":            {"url":hour1_url, "thumb":hour1_thumb, "type":"members"},
-               "Hour 2":            {"url":hour2_url, "thumb":hour2_thumb, "type":"members"},
-               "Aggressive Progressives" : {"url":ap_url, "thumb":ap_thumb, "type":"members"},
-               "The News with Dan Rather": {"url":danrather_url, "thumb":danrather_thumb, "type":"members"},
-               "Rebel Headquarters" : {"url":rebelhq_url, "thumb":rebelhq_thumb, "type":"members"},
-               "Post Game":         {"url":pg_url, "thumb":pg_thumb, "type":"members"},
-               "Old School":        {"url":oldschool_url, "thumb":oldschool_thumb, "type":"members"},
-               "TYT Classics":      {"url":tytclassics_url, "thumb":tytclassics_thumb, "type":"members"},
-               "Murder with Friends": {"url":murder_url, "thumb":murder_thumb, "type":"members"},
-               "Special Events":    {"url":special_events_url, "thumb":special_events_thumb, "type":"members"},
-               "Members LIVE show": {"url":members_live_url, "thumb":members_live_thumb, "type":"members_live"},
-               "Behind The Scenes": {"url":bts_url, "thumb":bts_thumb, "type":"members"}}
-              
-
-main_cat = {   "Members Only":      {"menu":"members", "thumb":hour1_thumb, "type":"members_menu"},
-               "TYT Sports":        {"url":tytsports_url, "thumb":tytsports_thumb, "type":"youtube_channel"},
-               "TYT Politics":      {"url":tytpolitics_url, "thumb":tytpolitics_thumb, "type":"youtube_channel"},
-               "What The Flick?!":  {"url":wtf_url, "thumb":wtf_thumb, "type":"youtube_channel"},
-               "Pop Trigger":       {"url":pop_url, "thumb":pop_thumb, "type":"youtube_channel"},
-               "TYT Interviews":    {"url":tytint_url, "thumb":tytint_thumb, "type":"youtube_channel"},
-               "Nerd Alert":        {"url":nerd_url, "thumb":nerd_thumb, "type":"youtube_channel"},
-               "Secular Talk":      {"url":sec_url, "thumb":sec_thumb, "type":"youtube_channel"},
-               "The Point with Ana Kasparian": {"url":point_url, "thumb":point_thumb, "type":"youtube_channel"},
-               "The Majority Report with Sam Seder": {"url":sam_url, "thumb":sam_thumb, "type":"youtube_channel"},
-               "TYT Live Stream":   {"url":tytlive_url, "thumb":tytlive_thumb, "type":"youtube_video"},
-               "ThinkTank":         {"url":thinktank_url, "thumb":thinktank_thumb, "type":"youtube_channel"}}
-menus = {"main":main_cat,"members":members_cat}
+MEDIA_URL = 'special://home/addons/{0}/resources/images/'.format(PLUGIN_ID)
```

Comment:
I thought resources/images are still used because of this.

Reply:
I didn't realize that line was in there.  I removed it on the commit I did 2 hours ago (After you mentioned the missing directory).